ecosystem/sf-indexer/aptos-substreams/src/lib.rs

```diff
@@ -85,26 +88,61 @@ fn block_to_block_output(input_block: Block) -> Result<BlockOutput, Error> {
     })
 }
 
-#[substreams::handlers::store]
-fn store_count(transaction: Transaction, store: store::StoreAddInt64) {
-    store.add(transaction.version, generate_trx_key(), 1);
-    store.add(
-        transaction.version,
-        generate_trx_type_key(transaction.r#type()),
-        1,
-    );
-}
-
-fn generate_trx_key() -> String {
-    String::from("total")
-}
+#[substreams::handlers::map]
+fn block_output_to_token(block: BlockOutput) -> Result<Tokens, Error> {
+    let mut tokens = vec![];
+    // let mut token_ownerships = vec![];
+    let mut token_datas = vec![];
+    let mut collection_datas = vec![];
+    for txn in block.transactions {
+        let txn_version = txn.transaction_info_output.unwrap().version;
 
-fn generate_trx_type_key(trx_type: TransactionType) -> String {
-    match trx_type {
-        TransactionType::Genesis => "genesis",
-        TransactionType::BlockMetadata => "block_metadata",
-        TransactionType::StateCheckpoint => "state_checkpoint",
-        TransactionType::User => "user",
+        if let Some(TxnDataOutput::User(_)) = &txn.txn_data {
+        } else {
+            continue;
+        }
```

Comment:
This should likely be a `if !matches(&txn.txn_data, Some(TxnDataOutput::User(_)))` statement, and I'm amazed that the linter didn't pick this up